tests/integration/select/select_core.js

```diff
@@ -431,7 +431,7 @@ QUnit.asyncTest( "dialog size select title should match the label when changed a
 
 			{
 				pagechange: {
-					src: $.mobile.pageContainer,
+					src: $( ".ui-pagecontainer" ),
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?